docs/AboutUs.md

```diff
@@ -7,48 +7,46 @@ We are a team based in the [School of Computing, National University of Singapor
 
 ## Project team
 
-### John Doe
+### Yu Zizhen
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/yuzizhen.png" width="200px">
 
-[[homepage](http://www.comp.nus.edu.sg/~damithch)]
-[[github](https://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](https://github.com/YuZizhen)]
 
-* Role: Project Advisor
+- Role: Developer
 
 ### Koh Guan Zeh
 
 <img src="images/kohguanzeh.png" width="200px">
 
 [[github](https://github.com/KohGuanZeh)]
 
-* Role: Developer
-* Responsibilities: Testing
+- Role: Developer
```

Comment:
I think your editor changed all the bullet points from `*` to `-`, lets keep it as `*` since that's what's used everywhere for now